### PR TITLE
#164046240 Integrating Travis CI to Authors Haven 

### PR DESCRIPTION
**Pull request by Etomovich**
**What does this PR do?**
This PR configures Travis CI and adds coverage and build badge to the README.md

**Description of task to be completed?**
- configure Travis CI
- configure Coveralls
- add coverage and build badges to README.md

**How should	this be manually tested?**
- [Travis CI](https://travis-ci.org/andela/ah-legion-backend/builds)

**Any background context you may want to provide?**
- Travis CI is a tool used to check builds and test of projects hosted in github. It also provides continous intergration of the project.

**What are the relevant pivotal tracer stories?**
- [#164046240](https://www.pivotaltracker.com/n/projects/2313280)
- [#164046243](https://www.pivotaltracker.com/n/projects/2313280)

### CODE REVIEW

**Comment by ElMonstro** [comment]
Add background context to PR message

File: .coveralls.yml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+service_name: travis-pro
+repo_token: bZWUyvMiRymHxVnOIP20PUAC2lt6vx7iZ
```
Comment (abulojoshua1) [inline .coveralls.yml]: @Etomovich, add an end of file here.

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  - pip install coverage
+
+# command to run tests
+script: 
+  	- python manage.py test
+    - coverage
+    
+after_success:
+  - coverallslanguage: python
+  - coveralls
```
Comment (abulojoshua1) [inline .travis.yml]: @Etomovich, add an end of file here.

File: .travis.yml [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+  
+# command to install dependencies
+install:
+  - pip install -r requirements.txt
+  - pip install coveralls
+  - pip install coverage
+
+# command to run tests
+script: 
+  	- python manage.py test
```
Comment (abulojoshua1) [inline .travis.yml]: Use the same indentation on all lines in this file.

**Comment by abulojoshua1** [comment]
@Etomovich, you are having another person's commits in this PR, please rebase well and remove those commits from your file.

File: .travis.yml [hunk 4]
```diff
@@ -0,0 +1,19 @@
+language: python
+python:
+  - "2.7"
```
Comment (abulojoshua1) [inline .travis.yml]: @Etomovich, narrow the tests to python 3.7 only.

**Comment by Etomovich** [comment]
@abulojoshua1 I have seen my error I raised a PR to master rather than develop sorry for the oversight.

**Review by ElMonstro — CHANGES_REQUESTED** [review]
Please fix this Error in travis build "Unable to download 3.7 archive. The archive may not exist. Please consider a different version.
". Consider changing the python version t0  3.6.

**Comment by MandelaK** [comment]
@Etomovich could you squash your commits into one? Also please try and make sure the build is passing, it could be a problem with your travis.yml configuration, but I am not so sure

**Review by cob04 — COMMENTED** [review]
Would you please squash your three commits into one.

**Comment by Etomovich** [comment]
> @Etomovich could you squash your commits into one? Also please try and make sure the build is passing, it could be a problem with your travis.yml configuration, but I am not so sure

Keith no test have been configured yet in this branch. I will expect the tests to be added before Travis can pass.

**Comment by Etomovich** [comment]
> Would you please squash your three commits into one.

I have squashed the changes, any more feedback will be highly appreciated.

**Comment by Etomovich** [comment]
> Please fix this Error in travis build "Unable to download 3.7 archive. The archive may not exist. Please consider a different version.
> ". Consider changing the python version t0 3.6.

I have downgraded it to python version 3.6. As per you recommendations. Thanks.

**Review by ElMonstro — APPROVED** [review]

**Review by BrianSerem — COMMENTED** [review]
Please input the link to Travis CI  as a way of testing instead of 'none'

**Comment by Etomovich** [comment]
> Please input the link to Travis CI as a way of testing instead of 'none'

Will do!!

**Pull request merged**